app/src/main/java/com/duckduckgo/app/widget/ui/WidgetCapabilities.kt

```diff
@@ -26,6 +26,7 @@ import com.duckduckgo.widget.SearchWidgetLight
 import javax.inject.Inject
 
 interface WidgetCapabilities {
+    val isAppWidgetSupported: Boolean
```

Comment:
Adding this method in case someone wants to use it on their implementation, but haven't touched every place using widget capabilities.